deny.toml

```diff
@@ -12,7 +12,6 @@ ignore = [
     "RUSTSEC-2020-0168", # mach dependency being unmaintained, dependency in consensus, we should consider moving to mach2 fork
     "RUSTSEC-2024-0370", # `cs_derive` needs to be updated to not rely on `proc-macro-error`
     # all below caused by StructOpt which we still use and we should move to clap v4 instead
-    "RUSTSEC-2024-0375",
```

Comment:
This was duplicated:
https://github.com/matter-labs/zksync-era/pull/3331/files#diff-1040309c64844eb1b6b63d8fd67938adbf9461f1b3c61f12cf738f064a02d3deR11